gunicorn/arbiter.py

```diff
@@ -501,6 +508,7 @@ def spawn_worker(self):
         pid = os.fork()
         if pid != 0:
             self.WORKERS[pid] = worker
+            self.WORKER_IDS[pid] = worker_id
```

Comment:
i don't understand why you need another dict? Why not simply adding an "id" to the worker instead?